parser/coldef/col_def.go

```diff
@@ -103,8 +103,11 @@ func getDefaultValue(c *ConstraintOpt, tp byte, fsp int) (interface{}, error) {
 
 		return value, nil
 	}
-
-	return expression.FastEval(c.Evalue), nil
+	v := expression.FastEval(c.Evalue)
+	if vv, ok := v.(*types.DataItem); ok {
```

Comment:
FastEval is used somewhere else. I am not sure about this. I am concerned  about the side effects.